es_ES/web-analogs.md

```diff
@@ -0,0 +1,900 @@
+---
+layout: page
+title: Flutter para Desarrolladores Web (HTML/CSS) 
+permalink: /web-analogs/
+---
+
+<link rel="stylesheet" href="/css/two_column.css">
+
+Contents:
+* TOC Placeholder
+{:toc}
+
+<div class="begin-examples"></div>
+Esta página es para usuarios familiarizados con la sintaxis HTML y CSS para la organización de componentes de UI de una aplicación. Al mapear fragmentos de código HTML/CSS a sus equivalentes de código Flutter/Dart.
```

Comment:
It maps HTML/CSS code snippets to their Flutter/Dart code equivalents.
Esta contrasta ejemplos de código en HTML/CSS con su código equivalente en Flutter/Dart